sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,31 +1,32 @@
 ## Release History
 
-### 4.71.0-beta.1 (Unreleased)
+### 4.71.0-beta.1 (2025-05-19)
 
 #### Features Added
+* Added Weighted RRF for Hybrid and Full Text Search queries - [PR 45328](https://github.com/Azure/azure-sdk-for-java/pull/45328)
 
-#### Breaking Changes
+### 4.70.0 (2025-05-16)
 
 #### Bugs Fixed
+* Fixed an issue where child partition is getting overridden with null continuation token if a split happens during the first request of a parent partition. - See [PR 45363](https://github.com/Azure/azure-sdk-for-java/pull/45363)
 
 #### Other Changes
+* Added a way to opt-in into Per-Partition Automatic Failover using enablement flag from the account metadata. - [PR 45317](https://github.com/Azure/azure-sdk-for-java/pull/45317)
 
-### 4.70.0 (2025-05-16)
+### 4.68.1 (2025-05-16)
+> [!IMPORTANT]
+> We strongly recommend customers with an intent to upgrade to version 4.70.0 or later to include this bug fix.
```

Comment:
Is this proper english? Honest question - itmight be just may incorrect usage :-) but I would expect something like 

We strongly recommend customers with an upgrade-intent to upgrade to version 4.70.0 or later to include this bug fix.

We strongly recommend customers with an intent to upgrade to upgrade to version 4.70.0 or later to include this bug fix.

We strongly recommend customers with an intent to upgrade to use version 4.70.0 or later to include this bug fix.

Reply:
@jeet1995 thoughts on above?

Reply:
I like the first version suggested by Fabian better -> 
We strongly recommend customers with an upgrade-intent to upgrade to version 4.70.0 or later to include this bug fix.

or we can simplify to : 

We strongly recommend customers to upgrade to version 4.70.0 or later to include this bug fix
?

Reply:
@FabianMeiswinkel, @kushagraThapar - I am good with any of the suggestions (I missed out the extra _upgrade to_) - main mindset was the customer shouldn't think 4.70.0 is now the new min recommended version, and that they absolutely have to upgrade. The customer if at all has the intent to upgrade can do so directly to 4.70.0 skipping 4.69.0.

Reply:
Got it, good suggestion, I will use the one with `upgrade-intent` wordings.

Reply:
FYI, I have updated it directly in the release branch, will also update it once the release is done to merge it back to main, thanks @jeet1995 and @FabianMeiswinkel